pkg/util/sys/device.go

```diff
@@ -121,6 +121,16 @@ type LocalDisk struct {
 	Encrypted bool `json:"encrypted,omitempty"`
 }
 
+// ListDevicesWithFilter list all devices available on a machine using the filterName to filter with grep
+func ListDevicesWithFilter(executor exec.Executor, filterName string) ([]string, error) {
+	devices, err := executor.ExecuteCommandWithOutput("lsblk", "--all", "--noheadings", "--list", "--output", "KNAME", "|", "grep", filterName)
```

Comment:
I don't believe you can pipe from this command. I'd suggest getting the output, then parsing in go. This way it will also be simpler to unit test.
```suggestion
devices, err := executor.ExecuteCommandWithOutput("lsblk", "--all", "--noheadings", "--list", "--output", "KNAME")
```

Reply:
Hi @travisn thanks for the review, I've updated my PR removing the pipe and now using parsing in go as you suggested